sponsorship_compassion/models/contracts.py

```diff
@@ -501,6 +501,9 @@ def create(self, vals):
 
         new_sponsorship = super().create(vals)
 
+        if new_sponsorship.partner_id and not new_sponsorship.correspondent_id:
+            new_sponsorship.correspondent_id = new_sponsorship.partner_id
+
```

Comment:
This was previously done in an `onchange` method, which I think is better, so that the user sees before saving that the correspondent is set. Otherwise he will think the correspondent is empty and will manually set it, which is what we want to avoid.